Update astro-support.md (#3909)
astronomer/docs: astro/astro-support.md

### P1: Critical impact

> A production Deployment is completely unavailable, or a DAG that was previously working in production is no longer working without having been modified.

Reviewer: ```suggestion
A production Deployment is completely unavailable, or a DAG that was previously working in production stops working, even though it was not changed.
```
